Description: fix FTBFS with GCC 14 with a special gRPC NULL value
 It contains a struct with should have a dimension / value, but a NULL value
 can't have that.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2024-08-11

---

Index: grpc/src/ruby/ext/grpc/extconf.rb
===================================================================
--- grpc.orig/src/ruby/ext/grpc/extconf.rb
+++ grpc/src/ruby/ext/grpc/extconf.rb
@@ -189,6 +189,7 @@ $CFLAGS << ' -std=c11 '
 $CFLAGS << ' -Wall '
 $CFLAGS << ' -Wextra '
 $CFLAGS << ' -pedantic '
+$CFLAGS << ' -Wno-incompatible-pointer-types '
 
 output = File.join('grpc', 'grpc_c')
 puts "extconf.rb $LDFLAGS: #{$LDFLAGS}"
